src/md/src/WorkWithReports/ViewLaunches.md

```diff
@@ -174,15 +174,20 @@ All the related data for the test case (logs, screenshots) will be saved on Log
 
 Log object has the following levels:
 
->   Error
```

Comment:
can we add some table with log levels and their int representations like

| Level        | Value           | 
| -------- |:-------------:|
| TRACE | 1000 |
| DEBUG | 2000 |
| INFO | 3000 |